### PR TITLE
blade template

### PR DESCRIPTION
**Pull request by huupt-0333**
- User List

image

- User Edit

image

- Task List

image

- Task Create

image

- Task Edit

image

### CODE REVIEW

**Comment by huupt-0333** [comment]
ready

File: resources/views/tasks/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                                    {{ __('Delete') }}
+                                </x-danger-button>
+                            </td>
+                        </tr>
+                    @endforeach
+                </tbody>
+            </table>
+        </div>
+    </div>
+</x-app-layout>
```
Comment (ghost) [inline resources/views/tasks/index.blade.php]: eof

File: resources/views/users/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                                    {{ __('Delete') }}
+                                </x-danger-button>
+                            </td>
+                        </tr>
+                    @endforeach
+                </tbody>
+            </table>
+        </div>
+    </div>
+</x-app-layout>
```
Comment (ghost) [inline resources/views/users/index.blade.php]: eof

**Comment by ghost** [comment]
Sửa xong merged luôn em nhé

**Pull request merged**